### ISSUE TITLE
[BUG] Icons occlude each other when KES is run in conjunction with subscriptions panel

### ISSUE THREAD

**Issue by aclist** [aclist/kbin-kes#73]
**Describe the bug**

The visual glitch also appears if I disable the Notifications Panel and Better Page Navigation scripts, so it seems to be something in the interaction between KES and Subscriptions Panel.

If it makes a difference, I use Firefox on Windows with Tampermonkey.

**KES version:**
- Version number 1.1.3

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Firefox

Cf. https://kbin.social/m/kbinMeta/t/183928/kbin-Enhancement-Suite-a-community-curated-script-manager-that-lets-you#entry-comment-741493

**Comment by aclist** [comment]
This is most likely related to behavior of the `nav-icons` mod

**Comment by Oricul** [comment]
This appears to be resolved (at least, in 1.2.0). I'm running LibreWolf though, a fork of Firefox. ![Quick Proof](https://files.catbox.moe/ufvlae.png)

**Comment by aclist** [comment]
There is actually a fourth icon that KSP tries to insert to the left of the three icons adjacent to your profile. Try forcing the viewport to mobile size and seeing if the "Kbin Subscriptions Panel" mobile icon appears to the left of the search icon.

**Comment by Oricul** [comment]
Yep, it pops the subscription panel to the left of the search icon _(note: this is the same functionality I had prior to updating to KES)_. ![Mobile View](https://files.catbox.moe/so3wpy.png)

**Comment by aclist** [comment]
I was able to reproduce the original bug in Firefox (I think it should affect most browsers, as the positional index of an element was being inserted incorrectly into the page CSS, because the KSP icon is `nth-of-type(1)`. I just wasn't able to reproduce whether it was working again, because KSP stopped functioning for me. I think this is fixed, let's wait for confirmation from @Prouvaire

**Comment by Prouvaire** [comment]
Thanks @aclist - glitch has been fixed: 
![KESglitchfixed](https://github.com/aclist/kbin-kes/assets/138431362/14f14855-f9bc-43a2-acbc-11efb71f28d0)
